mpc/coordinator/powersoftau_command.py

```diff
@@ -81,4 +80,9 @@ def _exec(
 
 
 def _default_powersoftau_path() -> str:
-    return join(dirname(__file__), "..", "..", "..", "powersoftau")
+    """
+    Return the default path to the PoT binaries directory
+    """
+    return join(
+        dirname(__file__),
```

Comment:
Did you mean to introduce this break?  All parameters fit on a single line (or if you prefer to break, each parameter should be on its own line to be consistent)

Reply:
Oh don't know why I broke the line here, you're right this is useless and inconsistent. Will move that up to the same line (same for below)

Reply:
Fixed in https://github.com/clearmatics/zeth/pull/266/commits/98bedfa67af3f884f53373ec5a403ffdf737bccc